apis/python/tests/test_tiledbobject.py

```diff
@@ -0,0 +1,42 @@
+import tiledbsoma as soma
```

Comment:
this is fantastic!

can you also add some non-string arguments?  bytes, dict, number, etc.

Reply:
@bkmartinjr the `uri` argument is of type `str`.

`soma.DenseNDArray.exists(123.45)` should, if anything, raise `ValueError` since 123.45 is not a valid URI in any sense.